packages/lab/src/list-next/ListItemNext.css

```diff
@@ -0,0 +1,61 @@
+/* Default variables applied to the root element */
+.saltListItemNext {
+  /* Color */
+  --listItemNext-text-color: var(--salt-text-primary-foreground);
+  --listItemNext-background: none;
+}
+
+/* Style applied to the root element */
+.saltListItemNext {
+  color: var(--listItemNext-text-color);
+  background: var(--listItemNext-background);
+  height: var(--salt-size-stackable, auto);
+  padding-inline: var(--salt-size-unit);
+  display: flex;
+  gap: var(--salt-size-unit);
+  position: relative;
+  align-items: center;
+  white-space: nowrap;
+  transition: background var(--salt-duration-perceptible) ease-in-out;
+}
+
+.saltListItemNext > * {
+  pointer-events: none;
+}
+
+/* Class applied to list item on focus using keyboard navigation only */
+.saltListItemNext-focused {
+  outline: var(--salt-focused-outline);
+  outline-offset: calc(var(--salt-focused-outlineWidth) * -1);
+}
+
+/* Active list item on hover using mouse */
+.saltListItemNext:hover:not([role="presentation"]):not(
```

Comment:
do we still have role=presentation in the code? We only cared for it when there used to be an empty message. Now that we removed it, I don't think this is necessary